src/main/scala/viper/gobra/ast/frontend/Ast.scala

```diff
@@ -42,8 +43,20 @@ sealed trait PUnorderedScope extends PScope
 case class PPackage(
                      packageClause: PPackageClause,
                      programs: Vector[PProgram],
-                     positions: PositionManager
+                     positions: PositionManager,
+                     id: String,
+                     isBuiltIn: Boolean = false
```

Comment:
I think it would be more readable to use `PackageInfo` here. This would also allow you to make `viperId` a `lazy val` of PackageInfo. This would imho be more natural instead of having this here